snuba/snuba_migrations/search_issues/0002_search_issues_add_tags_hash_map.py

```diff
@@ -0,0 +1,49 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Array, Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.storages.tags_hash_map import TAGS_HASH_MAP_COLUMN
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Copy of 0004_transactions_add_tags_hash_map.py
+
+    """
```

Comment:
Since `_tags_hash_map` is materialized from `tags.key` and `tags.value` I think this needs to be a separate migration after 0001 runs. Lemme know if there's a way to combine these two migrations into one.

Reply:
You could put the `AddColumn` operation after the `CreateTable` operation in your original migration. But leaving this separate seems a little cleaner, especially since `CreateTable` operations are pretty complicated.

Reply:
Ahh, right. Didn't realize I can have multiple operations in the same migration - that makes sense. I think I'll keep it as two though since it is cleaner.